### PR TITLE
using is_fixed

### PR DESCRIPTION
**Pull request by willtebbutt**
As per @oxinabox 's request in #26 .

I'm not sure how to test this as I'm not entirely sure what the problem was in the first place 🤷

### CODE REVIEW

**Comment by oxinabox** [comment]
Hopefully you can test it by adding what ever package triggered the problem as a test time dependency,
and then testing it much like the other tests in `test/activate_set.jl`

**Comment by willtebbutt** [comment]
Alas, the version of ChainRulesCore required in the test deps and KernelFunctions aren't yet compatible. Any other suggestions?

**Comment by oxinabox** [comment]
Change the version of ChainRulesCore to one that is compatible.
It just needs to be a version that has the same test time dependencies that we are checking for.
and then we lock it inplace with a `="=1.0.2"` to be sure.

**Comment by willtebbutt** [comment]
Okay -- I've not restricted CRC to 0.10.13, which is compatible with KernelFunctions 0.10.12. Is this alright? I'm still not really clear on how to know if this is a permissible version of CRC.

**Comment by willtebbutt** [comment]
Change of plan -- I've added a laughably simple test to prevent reversion, because I was unable to find a version of KernelFunctions which played nicely with ChainRulesCore on 1.4, 1.5, and 1.6, nor was I able to really tie down the cause of hitting that particular branch.

File: test/activate_set.jl [hunk 1]
```diff
@@ -39,4 +39,16 @@
...
         end
     end
+
+    # See #26
+    @testset "KernelFunctions" begin
+        orig_project_toml_path = Base.active_project()
+        push!(LOAD_PATH, mktempdir())  # put something weird in LOAD_PATH for testing
+        orig_load_path = Base.LOAD_PATH
+        try
+            TestEnv.activate("KernelFunctions")
```
Comment (oxinabox) [inline test/activate_set.jl]:
```suggestion
            TestEnv.activate("KernelFunctions")
            @test true   # would have errored before this point if this failed.
```

File: test/activate_set.jl [hunk 2]
```diff
@@ -39,4 +39,16 @@
             Pkg.activate(orig_project_toml_path)
         end
     end
+
+    # See #26
```
Comment (oxinabox) [inline test/activate_set.jl]: I prefer full URLs for comments so I can click them to go the other code
```suggestion
    # See https://github.com/JuliaTesting/TestEnv.jl/issues/26
```

File: test/activate_set.jl [hunk 3]
```diff
@@ -39,4 +39,16 @@
...
+    @testset "KernelFunctions" begin
```
Comment (oxinabox) [inline test/activate_set.jl]:
```suggestion
    @testset "KernelFunctions: is_fixed issue" begin
```

**Comment by oxinabox** [comment]
> I'm still not really clear on how to know if this is a permissible version of CRC.

It just needs to have the same test-time dependencies.

File: test/activate_set.jl [hunk 4]
```diff
@@ -39,4 +39,7 @@
             Pkg.activate(orig_project_toml_path)
         end
     end
+
+    # https://github.com/JuliaTesting/TestEnv.jl/issues/26
+    @test isdefined(TestEnv, :isfixed)
```
Comment (oxinabox) [inline test/activate_set.jl:44]: Welp, I guess that works.
Feels unsatisfying but, sure why not.

Comment (willtebbutt) [reply test/activate_set.jl:44]: Completely agree. Presumably the fact that it isn't hit in the existing tests means that a chunk of code doesn't get tested :(

**Comment by oxinabox** [comment]
lets merge this and later worry about finding a better test

**Review by oxinabox — APPROVED** [review]

**Pull request merged**